src/utils/utils.rs

```diff
@@ -341,7 +341,7 @@ where
     })
 }
 
-pub(crate) fn copy_file(src: &Path, dest: &Path) -> Result<()> {
+pub fn copy_file(src: &Path, dest: &Path) -> Result<()> {
```

Comment:
This is presumably being used from some test code; I'd like to keep the truely public functions as minimal as we can - just `fs::copy` is probably more than sufficient for the test suite case